components/divider/style/index.less

```diff
@@ -84,14 +84,29 @@
     border-width: @border-width-base 0 0;
   }
 
+  &-dotted {
+    background: none;
+    border-color: @divider-color;
+    border-style: dotted;
+    border-width: @border-width-base 0 0;
+  }
+
+  &-horizontal&-with-text&-dotted {
+    &::after,
+    &::before {
+      border-style: dotted none none;
+    }
+  }
+
   &-horizontal&-with-text&-dashed {
     &::before,
     &::after {
       border-style: dashed none none;
     }
   }
 
-  &-vertical&-dashed {
+  &-vertical&-dashed,
+  &-vertical&-dotted {
```

Comment:
Please move the newly added styles to patch.less, because they belong to antd5, zorro's current styles are synchronized from antd4 :)

Reply:
You're right :)
Your requested change is done 👍